packages/boot/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.tgz
+dist*
+package
```

Comment:
Can we use top-level `.gitgnore` from loopback-next monorepo and remove this file?